daml-script/test/src/test-utils/com/daml/lf/TestingAdminLedgerClient.scala

```diff
@@ -58,7 +58,7 @@ class TestingAdminLedgerClient(
             admin_topology_service.BaseQuery(
               filterStore = None,
               proposals = false,
-              operation = TopologyChangeOp.TOPOLOGY_CHANGE_OP_REPLACE_UNSPECIFIED,
+              operation = TopologyChangeOp.TOPOLOGY_CHANGE_OP_UNSPECIFIED,
```

Comment:
@moisesackerman-da @rgugliel-da can you please both check if this is the right move here?
for context the enum was changed recently from
```
  enum TopologyChangeOp {
    // Adds a new or replaces an existing mapping
    TOPOLOGY_CHANGE_OP_REPLACE_UNSPECIFIED = 0;

    // Remove an existing mapping
    TOPOLOGY_CHANGE_OP_REMOVE = 1;
  }
```
to
```
  enum TopologyChangeOp {
    TOPOLOGY_CHANGE_OP_UNSPECIFIED = 0;

    // Adds a new or replaces an existing mapping
    TOPOLOGY_CHANGE_OP_ADD_REPLACE = 1;

    // Remove an existing mapping
    TOPOLOGY_CHANGE_OP_REMOVE = 2;
  }
```

Reply:
Thanks for checking. No, you need to specify TOPOLOGY_CHANGE_OP_ADD_REPLACE